cmd/registry/cmd/compute/lint/lint.go

```diff
@@ -107,50 +119,27 @@ func lintRelation(linter string) string {
 }
 
 func (task *computeLintTask) Run(ctx context.Context) error {
-	spec := task.spec
-	err := visitor.FetchSpecContents(ctx, task.client, spec)
+	root, err := conformance.WriteSpecForLinting(ctx, task.client, task.spec)
+	if root != "" {
+		defer os.RemoveAll(root)
```

Comment:
Is it ok for our commands to blindly wipe out without an overwrite flag?

Reply:
This is removing a temporary directory, so I think it's ok here, but I could imagine a --debug flag that preserves them. Maybe a follow-up issue?